client/app/l10n/en.json

```diff
@@ -664,7 +664,7 @@
   "The specified comment was not found": "The specified comment was not found",
   "The specified host does not match any configured one": "The specified host does not match any configured one",
   "The specified input is not valid:": "The specified input is not valid:",
-  "The specified old password is not valid": "The specified old password is not valid",
+  "The specified old password is not valid": "The specified old password is no longer valid",
```

Comment:
Thanks you @comradekingu  but this correction misses context.
This is a texts shown when a user wants to change the password and it is requested the old password.

i will clarify this also on transifex for other translators.